sources/core/Stride.Core.Design/VisualStudio/VisualStudioVersions.cs

```diff
@@ -133,7 +134,7 @@ private static List<IDEInfo> BuildIDEInfos()
 
                         // Only deal with VS2019+
                         if (!Version.TryParse(inst2.GetInstallationVersion(), out var version)
-                            || version.Major < 15)
+                            || version.Major < 16)
```

Comment:
Even though the comment right above this if statement was updated, this check was not.  Note that in other places in the codebase, 2017 (15) is excluded from checks (see AllAvailableVisualStudioInstances above, for example) so for consistency, this method has been updated to only target 16 and greater.